controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -84,6 +85,8 @@ private class Delegate extends AbstractExecutionThreadService {
         private final EventProcessorConfig<T> eventProcessorConfig;
         private EventRead<T> event;
```

Comment:
I realize you didn't write this, but this does not need to be a member variable. It can just made a parameter to the `handleException` method as all of the calls are local so it can simply be passed as an argument.

Reply:
`handleException` method is being called from three places -> inside `startUp()` , `run()` and `restart()` . May be that is why `event` is made as member variable.